### ISSUE TITLE
E2E Test failures BuildIT.buildImage:111 » KubernetesClient Can't instantiate binary build

### ISSUE THREAD

**Issue by manusa** [fabric8io/kubernetes-client#5444]
## Description

Getting multiple failures (recently) with the buildImage test.

Haven't investigated it yet, but I'm worried that we might be running into timeout issues once again.

**Comment by shawkins** [comment]
> Haven't investigated it yet, but I'm worried that we might be running into timeout issues once again.

This does not appear related to timeouts, rather the instance seems to have insufficient disk - see the failed scheduling method: 

```
2023-09-12T12:13:55.1289374Z BuildStarted c48e4755755741ba8dfb4b735460dd08-bc-2.178425b49dccd769 Build myproject/c48e4755755741ba8dfb4b735460dd08-bc-2 is now running
2023-09-12T12:13:55.1290328Z Scheduled c48e4755755741ba8dfb4b735460dd08-bc-2-build.178425b45b59a2db Successfully assigned c48e4755755741ba8dfb4b735460dd08-bc-2-build to localhost
2023-09-12T12:13:55.1291348Z FailedScheduling c48e4755755741ba8dfb4b735460dd08-bc-1-build.178425b41e1c7a36 skip schedule deleting pod: myproject/c48e4755755741ba8dfb4b735460dd08-bc-1-build
2023-09-12T12:13:55.1292556Z BuildCancelled c48e4755755741ba8dfb4b735460dd08-bc-1.178425b41ed47e3c Build myproject/c48e4755755741ba8dfb4b735460dd08-bc-1 has been cancelled
2023-09-12T12:13:55.1293791Z FailedScheduling c48e4755755741ba8dfb4b735460dd08-bc-1-build.1784256e4683507f 0/1 nodes are available: 1 node(s) had disk pressure.
```

Also there haven't been any timeout related changes that I'm aware of.  Have their been changes to the test environment(s)?

**Comment by manusa** [comment]
> Also there haven't been any timeout related changes that I'm aware of. Have their been changes to the test environment(s)?

Everything has remained the same, that's why I was suspicious of an old bug resurfacing.

Maybe the GH environment now ships with more tools that take up space ~(we do spend a considerable amount of time removing unused packages for this same reason though)~

OK while writing the comment I just noticed that we removed the step to free up some space before execution #2276 (can't remember why - probably because GitHub was getting more generous). We might want to restore this step.
